### ISSUE TITLE
Images are no longer copied to dest.

### ISSUE THREAD

**Issue by boeckMt** [lumeland/lume#628]
### Version

- Deno@1.44.4 - lume@v2.2.2

### Platform

Windows

### What steps will reproduce the bug?

Put some images in a content folder next to the *.md File

change version of lume from `https://deno.land/x/lume@v2.2.0/` to `https://deno.land/x/lume@v2.2.1/` or `https://deno.land/x/lume@v2.2.2/` 

### What do you see instead?

Starting with Lume version `2.2.1` there are only the generated html files in dest, but no images anymore.

Is this a bug or intentional behavior? Can not find anything in the [CHANGELOG](https://github.com/lumeland/lume/blob/main/CHANGELOG.md) for this.

**Comment by oscarotero** [comment]
How are you copying the files? Can I see your _config.ts file or a repository reproducing this bug?

**Comment by haturatu** [comment]
add your `_config.ts`

```
site.copy("yourimagedir", "yourimagedir");
```
Check the path that the HTML is currently outputting after building, and change it to your image directory.

**Comment by boeckMt** [comment]
Hey @oscarotero this is my `_config.ts` file

```ts
import lume from "lume/mod.ts";
import base_path from "lume/plugins/base_path.ts";
import sass from "lume/plugins/sass.ts";
import picture from "lume/plugins/picture.ts";
import transformImages from "lume/plugins/transform_images.ts";

import esbuild from "lume/plugins/esbuild.ts";
import postcss from "lume/plugins/postcss.ts";
import { esbuildExternalFilter } from "./plugins/config.plugins.ts";

const theme = "clrCSS"; 

const site = lume({
    location: new URL('...'),
    src: "./src",
    dest: "./dist",
    components: {
        variable: "comp",
        cssFile: "/components.css",
        jsFile: "/components.js",
    }
});

//------------- Clarity styles - https://clarity.design/pages/developing ------------
const cssFiles = [
    { src: "https://unpkg.com/modern-normalize@1.1.0/modern-normalize.css", name: "modern-normalize.css" },
    { src: 'https://unpkg.com/@cds/core@6.12.0/global.min.css', name: 'global.min.css' },
    { src: 'https://unpkg.com/@cds/core@6.12.0/styles/theme.dark.min.css', name: 'theme.dark.min.css' },
    { src: 'https://unpkg.com/@clr/ui@17.0.1/clr-ui.min.css', name: 'clr-ui.min.css' }
];

cssFiles.forEach(item => {
    site.remoteFile(item.name, item.src);
});
//----------------------------------

site.ignore("README.md", "CHANGELOG.md", "node_modules");

site.use(base_path());
site.use(sass());

site.use(postcss());
site.use(picture());
site.use(transformImages());

//  Some JS and CSS files for an external library. 
site.copy("_static, ".");

site.use(esbuild({
    extensions: [".ts"],
    options: {
        bundle: true,
        format: "esm",
        minify: true,
        keepNames: true,
        ignoreAnnotations: true,
        platform: "browser",
        target: "esnext",
        treeShaking: true,
        external: ['https://esm.sh/v135/@cds/core@6.12.0/es2022/internal/interfaces/interfaces.js'],
        entryPoints: ["./src/clr-cds.ts"],
    }
}));

site.hooks.addEsbuildPlugin(esbuildExternalFilter);

export default site;

```

```json
{
  "tasks": {
    "lume": "echo \"import 'lume/cli.ts'\" | deno run -A -",
    "build": "deno task lume",
    "serve": "deno task lume -s",
  },
  "compilerOptions": {
    "types": [
      "lume/types.ts"
    ]
  },
  "imports": {
    "lume/": "https://deno.land/x/lume@v2.2.2/",
    "denoStd/": "https://deno.land/std@0.217.0/"
  }
}
```

**Comment by boeckMt** [comment]
@oscarotero thanks, I will try to check where the problem was and report it. With your project it works as expected.

**Comment by boeckMt** [comment]
@oscarotero I think I've figured out what the problem is. It was kind of a self-made issue because there was a _data.js file (no typings), and we used some properties of the page data, which changed.

There was a exported `url` function in a `_data.js` file to filter images for `data-pagefind-meta`.
The return was based on `page.src.asset` to return images or html files

```js
if (page.src.asset) {
      return `${url}${page.src.ext}`
 } else
      return `${url.replace(/\/index$/, '/index.html')
}`;
```

I found this change https://github.com/lumeland/lume/blob/ca04a1612db69fa5302ce6fe19148b3ae699eed8/core/utils/page_url.ts#L61 to https://github.com/lumeland/lume/blob/66dc59af843ab3ae0ce3bb4cc662299a38798ea8/core/utils/page_url.ts#L61

https://github.com/lumeland/lume/commit/396aee98b4dfa78d4c51897d9f33268f719d6251

I tried switching `_data.js` to `_data.ts` and using types. Now I got the error from the typings saying that this property doesn't exist. So, everything seems to be working as expected.
